enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/EditorButton.tsx

```diff
@@ -20,12 +21,12 @@ export const EditorButton = ({
       <ActionIcon
         data-testid="notebook-button"
         size="lg"
-        c={isOpen ? "text-white" : "text-dark"}
-        color="brand"
-        variant={isOpen ? "filled" : "default"}
         onClick={onClick}
+        className={S.EditorButton}
+        data-active={isOpen}
+        variant="default"
```

Comment:
why do we change variants here? Previously there was `filled` as well.

Reply:
@WiNloSt the filled variant interfered with the EditorButton CSS class. As I want complete control over the styles ( so they work in both dark and light mode), we use the default variant and handle the rest with CSS.